src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -92,6 +92,8 @@ public Task StartAsync(CancellationToken cancellationToken)
             {
                 if (_isSessionsEnabled)
                 {
+                    _logger.LogInformation($"The evaluated 'SessionHandlerOptions.AutoComplete' setting has '{_serviceBusOptions.SessionHandlerOptions.AutoComplete}' value");

```

Comment:
No need to log the AutoComplete setting here. We are logging all setting here:
https://github.com/Azure/azure-functions-servicebus-extension/blob/dev/src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Config/ServiceBusOptions.cs#L73

Reply:
I thought that just log whats there in the host.json? Since we updated this setting ServiceBusTriggerAttributeBindingProvider before creating the listener wont that change here? I think Mathew mentioned that in one of our conversations, I might be missing something. Could you confirm if the ones printed in the format would have the appropriate changed value as well?

Reply:
The service options format gets called before this update is made, so added a log while overriding the value.